admin_guide/install/prerequisites.adoc

```diff
@@ -295,7 +323,7 @@ dm.thinpooldev=/dev/mapper/docker--vg-docker--pool
 +
 [WARNING]
 ====
-This will destroy any docker containers or images currently on the host.
+This will destroy any Docker containers or images currently on the host.
 ====
```

Comment:
I think reinitializing Docker is more than this.  If I'm using option C and follow this docker will fail to start:

~~~
level=fatal msg="Shutting down daemon due to errors: error intializing graphdriver: Unable to take ownership of thin-pool (docker--vg-docker--pool) that already has used data blocks"
~~~

I've always had to forcefully clean up the stuff docker-storage-setup created:

~~~
# Assuming I had run this previously..
cat <<EOF > /etc/sysconfig/docker-storage-setup
DEVS=/dev/sdb
VG=docker-vg
SETUP_LVM_THIN_POOL=yes
EOF
docker-storage-setup

# I have to run this if /var/lib/docker/* is ever deleted.
lvremove docker-vg/docker-pool
pvremove --force --force /dev/sdb1
fdisk /dev/sdb
~~~